dmsg-daemon/cmd/internal/dmsg-client.go

```diff
@@ -0,0 +1,45 @@
+package internal
+
+import (
+	"context"
+	"time"
+
+	"github.com/skycoin/dmsg"
+	"github.com/skycoin/dmsg/cipher"
+	"github.com/skycoin/dmsg/disc"
+	"github.com/skycoin/skywire/pkg/skyenv"
+)
+
+// ConnClient connects to dmsg clients
+func ConnClient() error {
+	var snPK cipher.PubKey
+
+	// convert the pk from string to cipher.PubKey
+	err := snPK.Set(skyenv.DefaultSetupPK)
```

Comment:
We dont need to connect to the `setup node` here. This service is not a Skywire service and will not need to setup routes.

Reply:
I have used it only to test. Will change it after I implement the CSV logic.